src/MoBi.Presentation/Presenter/EditModuleConfigurationsPresenter.cs

```diff
@@ -154,7 +154,22 @@ public int CompareSelectedNodes(ITreeNode node1, ITreeNode node2)
          return _moduleConfigurationsDTOs.IndexOf(dto1) - _moduleConfigurationsDTOs.IndexOf(dto2);
       }
 
-      public IReadOnlyList<ModuleConfiguration> ModuleConfigurations => _moduleConfigurationsDTOs.Select(x => x.ModuleConfiguration).ToList();
+      public IReadOnlyList<ModuleConfiguration> ModuleConfigurations => _moduleConfigurationsDTOs.Select(mapNewModuleConfiguration).ToList();
+
+      private ModuleConfiguration mapNewModuleConfiguration(ModuleConfigurationDTO dto)
```

Comment:
Somehow I lost the generating of a new ModuleConfiguration from dtos during refactoring.